_includes/install/releasenotes/ee_install_20.md

```diff
@@ -10,7 +10,7 @@ See one of the following sections:
 ### Get Magento Commerce using Composer {#install-rc-composer}
 {:.no_toc}
 
-This software is available from `repo.magento.com`. Before installing the Magento Commerce software using Composer,  familiarize yourself with these  <a href="{{ page.baseurl }}/install-gde/prereq/integrator_install.html" target="_blank">prerequisites</a>, then run:
+This software is available from `repo.magento.com`. Before installing the Magento Commerce software using Composer,  familiarize yourself with these  <a href="{{ page.baseurl }}/install-gde/composer.html" target="_blank">prerequisites</a>, then run:
```

Comment:
Possible 2.0 using include? If used will 404 when clicked in 2.0 pages.

Reply:
Yes, a search for references to these includes in the 2.0 files did return results. I'll make changes.